src/data/datacube/seriestype.h

```diff
@@ -63,15 +73,20 @@ class SeriesType
 	}
 
 	std::string toString() const { return name; }
```

Comment:
if you use `std::string_view name;`, replace to `return {name.data(), name.size()};`

Reply:
done